src/sb/app/note/ui.cljc

```diff
@@ -15,23 +15,22 @@
             [sb.icons :as icons]
             [sb.routing :as routing]
             [sb.schema :as sch]
-            [sb.util :as u]))
+            [sb.util :as u]
+            [net.cgrand.xforms :as xf]))
 
 ;; TODO taken from project.ui/project-members check if can be merged
-(ui/defview note-members [project props]
+(ui/defview note-members [note props]
   ;; todo
   ;; 3. button for adding a new member via searching this board
   ;; 4. hover to see member details
   ;; 5. fix chat
   [:div.field-wrapper
    [:div.field-label (t :tr/members)]
    [:div.grid.grid-cols-2.gap-6
-    (for [member (->> (:membership/_entity project)
-                      (sort-by u/compare:desc :entity/created-at))
-          :let [board-membership (-> member :membership/member)
-                {:as account :keys [account/display-name]} (-> board-membership :membership/member)]]
+    (for [board-membership (member.data/members note (xf/sort-by :entity/created-at u/compare:desc))
```

Comment:
doesn't `member.data/members` return the member (eg. account) and not membership?

Reply:
I've made that switch in a different branch that sits on top of this one.